pkg/sql/logictest/testdata/logic_test/alter_primary_key

```diff
@@ -1715,3 +1706,60 @@ INSERT INTO t_multiple_cf VALUES (23, 24)
 
 statement ok
 ALTER TABLE t_multiple_cf ALTER PRIMARY KEY USING COLUMNS (j)
+
+subtest alter_primary_key_removes_rowid
+
+statement ok
+CREATE TABLE t_rowid (k INT NOT NULL, v STRING)
```

Comment:
To have the `SHOW CREATE` table statements below be stable, you'll need to explicitly set the column families. Another approach is to use `select column_name, column_id from crdb_internal.table_columns where descriptor_id = 't_rowid'::regclass` or something like that.

Reply:
Thanks! I struggled with this, all the more since `SHOW CREATE` in `cockroach demo` doesn't display families for some reason.

Reply:
It's not that demo doesn't display them, it's that the logictest randomize them in order to catch bugs.

Reply:
Oh, I get it now: and if there's just the 0 family, nothing's shown.